docs/user_guide/migrating_versions/migrating_to_v0_3.rst

```diff
@@ -0,0 +1,47 @@
+.. raw:: html
```

Comment:
The raw html won't be needed anymore (lines 1-6 and 8-13). Instead, "Onboarding guide for users ..." should be the header which in `.rst` is written as

```
Onboarding guide for users of version 0.2 or earlier
====================================================

Reply:
Actually, perhaps "Migrating to v0.3 from v0.2" is a more consistent title with the existing guide.

Reply:
done! Thanks